cmd/swapcli/suite_test.go

```diff
@@ -28,7 +31,10 @@ func TestRunSwapcliWithDaemonTests(t *testing.T) {
 	s.conf = daemon.CreateTestConf(t, tests.GetMakerTestKey(t))
 	t.Setenv("SWAPD_PORT", strconv.Itoa(int(s.conf.RPCPort)))
 	daemon.LaunchDaemons(t, 10*time.Minute, s.conf)
-	s.mockTokens = daemon.GetMockTokens(t, s.conf.EthereumClient)
+	ec := s.conf.EthereumClient.Raw()
+	pk := s.conf.EthereumClient.PrivateKey()
+	s.mockTether = contracts.GetMockTether(t, ec, pk)
+	s.mockDAI = contracts.GetMockDAI(t, ec, pk)
```

Comment:
This PR moves the mock tokens out of the `daemon` package and into the `contracts` package to make them available to more test packages without creating circular dependencies.